src/Contrib/Jaeger/AgentExporter.php

```diff
@@ -53,7 +53,7 @@ public function doExport(iterable $spans): int
     {
         // UDP Transport begins here after converting to thrift format span
         foreach ($spans as $span) {
-            $cSpan = $this->spanConverter->convert($span);
+            [$cSpan] = $this->spanConverter->convert([$span]);
```

Comment:
I thought cspan is a TV station, no?
Anyway, this could be one call:
```php
$this->jaegerTransport->append(
   $this->spanConverter->convert([$span]), 
   $this->serviceName
);
````

Reply:
Yeah I didn't quite understand the naming either...but in any case updated to one call instead